dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/BaseTestClass.java

```diff
@@ -298,9 +327,59 @@ private boolean validatePrometheusHTTPMetric(String vendorMetricsOutput, String
                 if (!line.startsWith("http_server_request_duration_seconds_count")) {
                     continue;
                 }
-                Log.info(c, "validatePrometheusHTTPMetric", "Potential match with: " + line);
+                Log.info(c, "validatePrometheusHTTPMetricCount", "Potential match with: " + line);
+                if (line.matches(matchString)) {
+                    Log.info(c, "validatePrometheusHTTPMetricCount", "Matched with: " + line);
+
+                    /*
+                     * If no custom count regex was supplied.
+                     * We will check if value is greater than 0
+                     */
+                    if (isDefaultCountCheck) {
+                        String[] split = line.split(" "); // should be only one space at the very end.
+                        assertEquals("Error. Expected 2 indexes from split " + Arrays.toString(split), split.length, 2);
+
+                        double countVal = Double.parseDouble(split[1].trim());
+                        assertTrue("Expected count value to be greater than 0", countVal > 0);
+                    }
+
+                    return true;
+                }
+            }
+        }
+
+        return false;
+    }
+
+    private boolean validatePrometheusHTTPMetricSum(String vendorMetricsOutput, String route, String responseStatus, String requestMethod, String errorType, String count,
```

Comment:
Line 393 (I can't add a comment at that line), should read "Expected sum value to be greater than 0"